index.html

```diff
@@ -33,9 +33,149 @@
      <a href="https://www.google.com"><i class="fa fa-google"></i></a>
     </li>
     </ul>
+   
+        <img class="seebutton" src="./Disabled.png" alt="button">
+    
 </div>
 </section>
-
+<section>
+    <div class="second-header">
+        <h2 class="second">My Recent work</h2>
+        <img src="./Indicator.png" alt="indicator">
+    </div>
+    <div class="grid-container">
+        <div class="grid-item first">
+            
+           <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+           </div>
+       
+        <div class="grid-item third">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
```

Comment:
- [ ] you also didn't follow the figma design template with the fonts, padding and line-height for these text. please make sure to go over them as provided in figma and apply them appropriately to your design beofre making your next review :+1: 
- figma design
![f-fonts](https://user-images.githubusercontent.com/89055502/235299248-2f778075-bcdc-4c0b-b5cb-6b43a40e8bd6.png)

Reply:
Hi @Efiamotu-1 ,
I am unable to change the fonts. when i am trying to make changes I got error in stylelintrc.